databind/src/databind/json/tests/converters_test.py

```diff
@@ -713,3 +713,25 @@ def of(cls, v: str) -> "MyCls":
     mapper = make_mapper([JsonConverterSupport()])
     assert mapper.serialize(MyCls(), MyCls) == "MyCls"
     assert mapper.deserialize("MyCls", MyCls) == MyCls()
+
+
+UnboundGeneric = t.TypeVar("UnboundGeneric")
```

Comment:
```suggestion
UnboundTypeVar = t.TypeVar("UnboundTypeVar")
```

Since the term `Generic` is used for the class and not the type variable, I find this more accurate.